packages/sitecore-jss/src/i18n/mockAppRootQueryResponse.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
I agree that this is a good idea, and I was thinking of doing this. I want to do this as part of a separate refactoring because I was to look at all the tests in sitecore-jss and see which tests can utilize this data. I have added this to my refactoring list in PBI 467457

Reply:
Agree, thank you